inc/testrunnerswitcher.h

```diff
@@ -20,6 +20,9 @@
 
 typedef void* TEST_MUTEX_HANDLE;
 
+// a macro useful for disabling tests while debugging
+#define TEST_DISABLED_FUNCTION(name)    void name(void)
```

Comment:
>TEST_DISABLED_FUNCTION [](http://example.com/codeflow?start=8&length=22)

VIM users would prefer DISABLED_TEST_FUNCTION.  Inserting/removing at the beginning of a line/word is less taxing. Not really a big deal, just an I-woulda-done-it-differently comment.

Reply:
something something emacs :P

Reply:
Ty hun, "happy wife, happy life".